Tests/BugsnagTests/Mocks/ConfigurationMock.swift

```diff
@@ -3,10 +3,17 @@ import Bugsnag
 
 internal class ConfigurationMock: ConfigurationType {
     let apiKey = "1337"
-    let notifyReleaseStages: [String] = []
+    let notifyReleaseStages: [String]?
     let endpoint = "some-endpoint"
     let filters: [String] = ["someFilter"]
 
-    required init(drop: Droplet) throws {}
-    init() {}
+    required init(drop: Droplet) throws {
```

Comment:
Since `notifyReleaseStages` now are optional, we could remove the first two initializers as I see it - what do you think?